src/svgdom/util/elements_by_class_name_finder.hpp

```diff
@@ -0,0 +1,30 @@
+#pragma once
+
+#include <unordered_map>
+
+#include "../elements/element.hpp"
+
+#include "style_stack.hpp"
+
+namespace svgdom{
+
+class elements_by_class_name_finder{
+public:
+
+	elements_by_class_name_finder(const svgdom::element& root);
+
+	const std::vector<const element*> find(const std::string& cls)const;
```

Comment:
I think this method should return reference to vector to avoid copying, i.e. `const std::vector<const element*>&`